src/common/function/src/scalars/math.rs

```diff
@@ -44,6 +44,8 @@ impl MathFunction {
         registry.register(Arc::new(RateFunction));
         registry.register(Arc::new(RangeFunction));
         registry.register(Arc::new(ClampFunction));
+        registry.register(Arc::new(ClampMinFunction));
+        registry.register(Arc::new(ClampMaxFunction));
```

Comment:
Can we use SQL functions `GREATEST()` and `LEAST()` to replace them?

Reply:
Yes, they are already supported by DataFusion, maybe we can just alias them.

Reply:
they have different signatures